libraries/chain/webassembly/runtimes/eos-vm-oc.cpp

```diff
@@ -51,7 +55,11 @@ class eosvmoc_instantiated_module : public wasm_instantiated_module_interface {
 };
 
 eosvmoc_runtime::eosvmoc_runtime(const std::filesystem::path data_dir, const eosvmoc::config& eosvmoc_config, const chainbase::database& db)
-   : cc(data_dir, eosvmoc_config, db), exec(cc), mem(wasm_constraints::maximum_linear_memory/wasm_constraints::wasm_page_size) {
+   : cc(data_dir, eosvmoc_config, db) {
+   for (auto i = 0; i < 14; ++i) {
```

Comment:
This will be changed to use lockfree stack pool.

We cannot do default 16, which results in mmap failure.

Reply:
Fixed by latest refactoring.